src/commands/apps/create.ts

```diff
@@ -93,7 +93,7 @@ export default class AppCreateCommand extends Command {
     debug(`Creating app with attributes: ${inspect(appAttributes)}`);
     const createAppResponse = await out.progress(
       "Creating app ...",
-      clientRequest<models.AppResponse>((cb) => client.apps.create(appAttributes, cb))
+      clientRequest<models.AppResponse>((cb) => client.appsOperations.create(appAttributes, cb))
```

Comment:
why this change?

Reply:
This is strange, looks like a breaking change? One we can handle, of course, as you did.

Reply:
I don't know why it changed some of the operations. It could be some new tag in our swagger. I doubt it would be a change from autorest itself since we're using the same major version.